tests/test_embedding.py

```diff
@@ -0,0 +1,546 @@
+import unittest
+import itertools
+from collections import Mapping
+import random
+
+import numpy as np
+import numpy.testing as npt
+
+import dimod
+
+try:
+    import networkx as nx
+    _networkx = True
+except ImportError:
+    _networkx = False
+
```

Comment:
Since `networkx` is required by a large number of tests below (almost half) which test the basic functionality of `dimod`, I would just put it as a requirement for running tests, and got rid of all `skipUnless` decorators. Unless there's some scenario I'm missing here.

Reply:
I like re-running the tests in a clean venv using `pip install .` as a check on the version ranges.